--- light-1.4.6/src/moz_main.cpp.orig	Wed Jul 18 07:53:02 2001
+++ light-1.4.6/src/moz_main.cpp	Thu Jul 26 16:52:56 2001
@@ -605,7 +605,7 @@
   if (!doc)
     return NULL;
 
-#ifdef MOZ_CVS
+#if 1
   nsCOMPtr <nsIURI> uri;
   doc->GetDocumentURL(getter_AddRefs(uri));
 #else
@@ -643,7 +643,7 @@
   if (!doc)
     return NULL;
 
-#ifdef MOZ_CVS
+#if 1
   nsCOMPtr <nsIURI> uri;
   doc->GetDocumentURL(getter_AddRefs(uri));
 #else
